simulate/simulate.go

```diff
@@ -233,7 +233,7 @@ func MutateGene(inputSeq []dna.Base, branchLength float64, geneFile string, dele
 	}
 	mutatedSequence := BaseExtToBases(newSequence)
 	delFound := seqContainsDels(mutatedSequence)
-	if delFound == false && deletions == true {
+	if !delFound {
```

Comment:
It was a while since i've looked at this, but I think the first time i read it though I processed it as "if deletions are found and deletions are true" which both sound true. Proceeding that I then realized it was "not found" and added back the "!" and now I see that deletion is a parameter that is passed in

Reply:
ok, can you change this to "if !delFound && deletions {" ?

Reply:
https://github.com/vertgenlab/gonomics/blob/6d0053a71b31c29892497681dd68429d134d2ebb/simulate/simulate.go#L236

Reply:
Sound be reflected in commit https://github.com/vertgenlab/gonomics/commit/6d0053a71b31c29892497681dd68429d134d2ebb